src/resolvers/acl.js

```diff
@@ -1,5 +1,8 @@
 import { parseResolveInfo } from 'graphql-parse-resolve-info'
+import camelcaseKeys from 'camelcase-keys'
+import { isEmpty } from 'lodash'
```

Comment:
We've not switched to `lodash-es` yet, so for now these imports should be `import isEmpty from 'lodash/isEmpty'`